tests/compat/test_client.py

```diff
@@ -429,16 +433,22 @@ def run_command(
         port=None,
         password=None,
         key_filename=None,
+        protocol_version=None,
     ):
         cmd, path = command.split(" ")
         cmd = cmd.split("-", 1)
         path = path.replace("'", "")
+        env = copy.deepcopy(os.environ)
```

Comment:
Does this need a deep copy rather than just dict(os.environ)?

Reply:
It works with just dict(), indeed.  I don't recall specifics for why I made it use deepcopy() . I recall seeing some problems with existing environment being modified at some point somehow.

The HTTP server tests also use a deepcopy() function. Perhaps I just copied from there.